CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## [1.0.1] - 2017-08-09
 ### Fixed
 - check-ceph.rb: fixed a bug where the ignore flag was not working due to order of operations (@jklare)
+- metrics-ceph[-osd].rb: fixed invalid call to JSON.parse
```

Comment:
this needs to be moved up under the unreleased section I will fix this up post acceptance for you.

Reply:
My bad, totally but this in the wrong place.
Sorry about that and thanks for fixing it!